config/crds/machineset.crd.yaml

```diff
@@ -94,6 +94,22 @@ spec:
           - replicas
           type: object
   version: v1alpha1
+  additionalPrinterColumns:
+  - JSONPath: .spec.replicas
+    description: Desired Replicas
+    name: Desired
+    type: string
+  - JSONPath: .status.replicas
+    description: Current Replicas
+    name: Current
+    type: string
```

Comment:
Same question as on the other PR. Should these be `type: integer`?

Reply:
Bumped to integers.